DQN/environment.yml

```diff
@@ -0,0 +1,35 @@
+name: mario
```

Comment:
Add matplotlib as dependency

Reply:
When I install matplotlib it says it requires pyglet<=1.5.0,>=1.4.0, although yml file installs pyglet 1.5.5, incompatible